refinery/core/models.py

```diff
@@ -1873,11 +1880,8 @@ def get_managed_group(self):
             return None
 
     def save(self, *args, **kwargs):
-        if len(self.name) == 0:
-            logger.error("Group name cannot be empty.")
-            return
-        else:
-            super(ExtendedGroup, self).save(*args, **kwargs)
+        self.name = self.clean()
```

Comment:
https://docs.djangoproject.com/en/1.7/ref/models/instances/#django.db.models.Model.full_clean - "Note that full_clean() will not be called automatically when you call your model’s save() method. " That's only an issue with ModelForm. With ModelForms, the clean() methods do return cleaned_data according to Two Scoops 11.7.